docs/src/pages/components/data-grid/columns/columns.md

```diff
@@ -34,43 +34,47 @@ By default, columns are ordered according to the order they are included in the
 > Otherwise, you take the risk of losing the column width state (if resized).
 > You can create the array outside of the render function or memoize it.
 
-## Column headers
+### Title & Description
 
 You can configure the headers with:
 
 - `headerName`: The title of the column rendered in the column header cell.
 - `description`: The description of the column rendered as tooltip if the column header name is not fully displayed.
-- `hide`: Hide the column.
-
-{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline"}}
 
-For more advanced header configuration, go to the [rendering section](/components/data-grid/rendering/#header-cell).
+{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline", "defaultCodeOpen": false}}
```

Comment:
It seems important for developers to have has many inline-previews as possible:

```suggestion
{{"demo": "pages/components/data-grid/columns/HeaderColumnsGrid.js", "bg": "inline"}}
```

What we have on HEAD, it's clear, it closes doors for confusion:

<img width="860" alt="Screenshot 2021-04-08 at 11 29 01" src="https://user-images.githubusercontent.com/3165635/114003226-dd78e400-985d-11eb-809d-19fb2c488d10.png">


x the other demos